### ISSUE TITLE
Laravel Telescope default config file throws error

### ISSUE THREAD

**Issue by rognales** [imanghafoori1/laravel-microscope#51]
My stack as follows:-

PHP 7.3
Laravel Framework 7.10.2
laravel-microscope v1.0.34

Errors reported by Laravel Microscope:-

````php
Class does not exist:
Laravel\Telescope\Watchers   <==== does not exist
at config\telescope.php:113
````

16 errors all together comes from ```telescope.php```

Thx.

**Comment by imanghafoori1** [comment]
Do they really exist? 
Is the telescope package installed?

**Comment by rognales** [comment]
Yes. I can access /telescope

**Comment by imanghafoori1** [comment]
It seems that require-dev classes is not detectable by PHP .
The class_exists() function just returns false. 
I think if you require the telescope as a normal dependency it would be fine.

**Comment by egorsmkv** [comment]
@imanghafoori1 they are detectable because they are in the vendor folder.

I just made telescope as a normal dependency and the issue still exists.

**Comment by egorsmkv** [comment]
A workaround for the issue is to create `config/microscope.php` with the following settings:

```
<?php

return [
    /**
     * An array of patterns that should be ignored for the checks.
     */
    'ignore' => [
        'config/telescope.php'
    ],
];
```

**Comment by imanghafoori1** [comment]
It is even more mysterious !!!

**Comment by egorsmkv** [comment]
Also, Telescope adds more mess:

```
yehor@desktop:~/Work/YehorSmoliakov/github/laravel-api-example$ php artisan check:events -vvvvvvv
Checking events...

1 errors found for events
|                                                                                                       |
|  1 Incorrect namespace.                                                                               |
|  ['namespace' => 'Laravel\Telescope\Http\Controllers'] passed to Route::group(...) is not correct.    |
|                                                                                                       |
|*******************************************************************************************************|  
Total elapsed time: 0.01 sec
 - 33 listenings were checked.
```

**Comment by egorsmkv** [comment]
My workaround is this: just disable Telescope before running laravel-microscope:

```
TELESCOPE_ENABLED=false php artisan check:events
```

**Comment by imanghafoori1** [comment]
this should be fixed as of version v1.0.122 
Thanks to everybody.
